docs/resources/Application.md

```diff
@@ -4,32 +4,35 @@
+| rpc_origins?                       | array of strings                                                           | List of RPC origin URLs, if RPC is enabled                                                                            |
+| bot_public                         | boolean                                                                    | when `false`, only the app owner can add the app to guilds                                                            |
+| bot_require_code_grant             | boolean                                                                    | when `true`, the app's bot will only join upon completion of the full OAuth2 code grant flow                          |
+| bot?                               | partial [user](#DOCS_RESOURCES_USER/user-object) object                    | Partial user object for the bot user associated with the app                                                          |
+| terms_of_service_url?              | string                                                                     | URL of the app's Terms of Service                                                                                     |
+| privacy_policy_url?                | string                                                                     | URL of the app's Privacy Policy                                                                                       |
+| owner?                             | partial [user](#DOCS_RESOURCES_USER/user-object) object                    | Partial user object for the owner of the app                                                                          |
+| summary *(deprecated)*             | string                                                                     | **deprecated and will be removed in v11.** An empty string.                                                           |
+| verify_key                         | string                                                                     | Hex encoded key for verification in interactions and the GameSDK's [GetTicket](#DOCS_GAME_SDK_APPLICATIONS/getticket) |
+| team                               | ?[team](#DOCS_TOPICS_TEAMS/data-models-team-object) object                 | If the app belongs to a team, this will be a list of the members of that team                                         |
+| guild_id?                          | snowflake                                                                  | Guild associated with the app. For example, a developer support server.                                               |
+| guild?                             | partial [guild](#DOCS_RESOURCES_GUILD/guild-object) object                 | Partial object of the associated guild                                                                                |
```

Comment:
AFAIK, the `guild` field is only present if the support server is DISCOVERABLE. Maybe this can be mentioned here?

Reply:
That's correct.